MY_README.md

```diff
@@ -0,0 +1,27 @@
+# Takeaway App
+
+To run this app you will first need to setup an account with [Twillio](www.twilio.com). After you have setup a number to text to and from you will need to put the following information into a `.env` file in the route of this project:
+
+```ACCOUNT_SID=account_sid
+AUTH_TOKEN=auth_token
+MESSAGING_SERVICE_SID=messaging_service_sid
+RECEIVER_NUMBER=receiver_number
+```
+
```

Comment:
Maybe add a message here to explain how to run the app. 
 For example

```
irb -r ./lib/takeaway.rb
```

Reply:
You could also explain basic commands that could be used to control it.

Reply:
You could also mention that you need to run 

```
bundle install
```

so that I have all required gems (e.g. pry) for running RSpec tests